spec/delivery_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rspec'
+require_relative '../lib/dependencies'
+
+
+RSpec.describe DeliveryService do
+  let(:service) { DeliveryService.new(3,60) }
```

Comment:
use `subject` for variables what correspond testable class

Reply:
tests rewritten